index.html

```diff
@@ -13,17 +13,20 @@
 
     <script>
         var calculator = {
-            addNumbers: function () {
+            addNumbers: function () 
                 // hint: these are strings
                 var numberOne = document.forms.inputs.numberOne.value;
                 var numberTwo = document.forms.inputs.numberTwo.value;
                 document.forms.inputs.result.value = numberOne + numberTwo;
-            },
-            subtractNumbers: function () {
-                
             }
-        }
 
+            var calculator2 = {
```

Comment:
The need for a new calculator object was not required. There is no harm in it though for the case of this work.